### ISSUE TITLE
PR_CONNECT_RESET_ERROR

### ISSUE THREAD

**Issue by MarcSN311** [Ysurac/openmptcprouter#1971]
## Expected Behavior
Using openmptcprouter should not break the internet connection

## Current Behavior
- When opening a Website in Firefox I get PR_CONNECT_RESET_ERROR
- Chrome also doesn't work

## Specifications

  - OpenMPTCProuter version: 0.58.5
  - OpenMPTCProuter VPS version: 0.58.5
  - OpenMPTCProuter platform: x86_64
 
![Screenshot (567)](https://user-images.githubusercontent.com/12860202/126485209-3b526dfc-f6ef-4596-b597-bd65a46c34ff.png)

**Comment by Ysurac** [comment]
You have a very high usage on the VPS. Check via htop for example what is using so much CPU.
You get the error for all websites ?

**Comment by MarcSN311** [comment]
It is a Hetzner root server with i7-4770 and 32GB memory, that also had a few docker containers running.
I get the error for all websites, doesn't look like IP blocking or something like that.

**Comment by Ysurac** [comment]
Check log on the VPS, in /var/log/daemon.log on Debian.

**Comment by MarcSN311** [comment]
Nothing in there looks suspicious to me. Only some entries like this
```
Jul 21 15:47:42 hive v2ray[668]: 2021/07/21 15:47:42 127.0.0.1:15317 accepted tcp:127.0.0.1:0 [api]
```
While I'm testing.

**Comment by Ysurac** [comment]
By default shadowsocks is used as proxy, so it's not related.
```curl https://1.1.1.1/``` via SSH on the router is working ?

**Comment by MarcSN311** [comment]
yeah curl on the router seems to work. looks like there is something wrong between my computer and the openmptcprouter.

**Comment by Ysurac** [comment]
No IPv6 on client computer ? DNS and gateway are OpenMPTCProuter IP ? No other DHCP server on network ? No other box/firewall/router/AP/... between client computer and OpenMPTCProuter ? You tested with another browser ?

**Comment by MarcSN311** [comment]
- [x] No IPv6 on client computer ?
- [x] DNS and gateway are OpenMPTCProuter IP ?
- [ ] No other DHCP server on network ?
- [x] No other box/firewall/router/AP/... between client computer and OpenMPTCProuter ?
- [x] You tested with another browser ?

I do have another DHCP on the network, but I blacklisted my test devices MAC on that one, so it gets its config from OpenMPTCProuter.

I moved OpenMPTCProuter and my test device to a new VLAN and now it works. Not sure if it was related to DHCP (why?!) or what else on the other vlan interferes with my connection but I will find it.

Thanks for your help and for a great project.
